frontend/src/modules/organization/components/list/organization-common-list-table.vue

```diff
@@ -1,79 +1,85 @@
 <template>
   <div>
-    <lf-table type="bordered" class="pb-12">
+    <lf-table class="pb-12">
       <thead>
         <tr>
-          <lf-table-head v-model="sort" property="displayName" :sticky="true">
-            ORGANIZATION
+          <lf-table-head v-model="sort" property="displayName" class="pl-3">
+            Organization
           </lf-table-head>
           <lf-table-head>
-            ASSOCIATED PROJECT GROUPS
+            Project groups
           </lf-table-head>
           <lf-table-head>
-            IDENTITIES
+            Identities
           </lf-table-head>
-          <lf-table-head :sticky="true" />
+          <lf-table-head />
         </tr>
       </thead>
 
       <tbody>
-        <tr v-for="(org, oi) of props.organizations" :key="org.id">
-          <lf-table-cell :sticky="true">
+        <router-link
+          v-for="(org, oi) of props.organizations"
+          :key="org.id"
+          class="table-row tr hover:bg-gray-100 transition h-16 max-h-16"
+          :to="{ name: 'organizationView', params: { id: org.id } }"
+        >
+          <lf-table-cell class="pl-3">
```

Comment:
I think I see 2 issues with the routes:
(1) The Organizations route is selected even tho it's disabled
(2) The back button on the Organization profile header, is returning to Admin Panel but not to the Organizations tab. It's defaulting to the first tab.

<img width="577" alt="Screenshot 2024-08-21 at 12 19 54" src="https://github.com/user-attachments/assets/52cc2f7b-62f2-4d60-8944-f92d70cd9500">